modules/local/wipertools/gather.nf

```diff
@@ -0,0 +1,43 @@
+process GATHER {
+    tag "$meta.id"
+    label 'process_single'
+    container 'docker.io/mazzalab/fastqrepair_nf_env:1.0.1'
```

Comment:
Is there a possibility to use a seqera container and/or biocontainers? or submit to bioconda?